src/libraries/System.Runtime.InteropServices/gen/DllImportGenerator/Diagnostics/Events.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Diagnostics.Tracing;
+
+namespace Microsoft.Interop.Diagnostics
+{
+    [EventSource(Name = "Microsoft-Interop-SourceGeneration-Events")]
+    internal sealed class Events : EventSource
```

Comment:
Are these still needed?  Weren't similar events added to the Roslyn infrastructure for all generators?